templates/rhdm70-full.yaml

```diff
@@ -67,6 +67,11 @@ parameters:
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
+- displayName: KIE Server ID
+  description: KIE execution server identifier. Corresponds to template ID defined by the controller instance. (Sets the org.kie.server.id system property)
```

Comment:
Sounds good tome, just add in the description telling that the kie.server.id will be set auto populated using the HOSTNAME var or a randin id.

Plus, add this new var in the kie-server image.yaml for rhdm and rhba for doc purposes.

Reply:
Here is a rewrite of the description that I would suggest.

Decision server identifier. Determines the template ID in Decision Central or controller. If this parameter is left blank, it is set using the $HOSTNAME environment variable or a random value. (Sets the org.kie.server.id system property).

Reply:
Description updated, thanks for the text.